app/templates/home.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html'%}
+
+{% block title%}
+    Home
+{% endblock%}
+
+{% block content %}
+<!-- CONTENT HERE -->
+<div class="container">
+    <p>Welcome to Batteryistic, with Battery Energy Storage Systems, you can start saving and thriving</p>
+</div>
+<div class="container">
+    <button class="btn btn-outline btn-success">Start Saving</button>
```

Comment:
This is more of a future improvement, but maybe clicking this button can be used to display the cost-saving table below, instead of showing it from the start.